README.md

```diff
@@ -113,6 +113,7 @@ http {
                 Expiration leeway for access_token renewal.
                 If this is set, renewal will happen access_token_expires_leeway seconds before the token expiration.
                 This avoids errors in case the access_token just expires when arriving to the OAuth Resoource Server.
+             --force_reauthorize = true
```

Comment:
I didn't see the other examples were using the defaults, sorry. Will open a new PR to fix it.